third_party/ti_simplelink_sdk/mcuboot/mcuboot.syscfg

```diff
@@ -73,20 +77,30 @@ else
 CCFG.ccfgTemplate.$name      = "ti_devices_CCFG_CCFGCC26XXTemplate0";
 
 /* ======== GPIO ======== */
-var GPIO = scripting.addModule("/ti/drivers/GPIO");
 
-var gpio0 = GPIO.addInstance();
-gpio0.$hardware = system.deviceData.board.components.LED0;
-gpio0.$name = "CONFIG_GPIO_LED_0";
+Button1.$name                    = "CONFIG_BTN_LEFT";
```

Comment:
What are the thoughts about custom HW? Not all implementations might have buttons or LEDs 🤔

Reply:
In that case, you should be able to remove the Button/LED modules and assignments from the mcuboot.syscfg file. In addition, you can add the define EXCLUDE_GPIOS when building the MCUBoot image.